modules/infra/control/worker.c

```
@@ -222,16 +222,29 @@ int port_plug(struct iface_info_port *p) {
-	for (unsigned cpu = 0; cpu < CPU_SETSIZE; cpu++) {
-		if (CPU_ISSET(cpu, affinity)) {
-			if (cpu == cpu_id)
-				break;
-			txq++;
+static void assign_txqs(gr_vec struct iface_info_port **ports) {
+	struct iface_info_port *port;
+	struct worker *worker;
+
+	// clear all TX queue assignments
+	STAILQ_FOREACH (worker, &workers, next)
+		gr_vec_free(worker->txqs);
```

Cppcheck (2.19.0)

[error] Expression 'worker->txqs?free(__gr_vec_hdr(worker->txqs))

(unknownEvaluationOrder)